yt_dlp/extractor/pornhub.py

```diff
@@ -325,6 +346,7 @@ def dl_webpage(platform):
                 })
             thumbnail = flashvars.get('image_url')
             duration = int_or_none(flashvars.get('video_duration'))
+            chapters = self._extract_chapters_from_action_tags(flashvars.get('actionTags'), duration)
```

Comment:
you could inline this here using `traverse_obj`
```suggestion
            chapters = traverse_obj(flashvars, (
                'actionTags', {lambda x: x.split(',')}, ..., {lambda x: x.split(':', 1)},
                all, lambda _, v: int_or_none(v[1]) is not None, {
                    'title': (0, {str.strip}),
                    'start_time': (1, {int_or_none}),
                })) or None
```

Reply:
fixed in [1565a11](https://github.com/yt-dlp/yt-dlp/pull/13685/commits/1565a11e9a1229f60b859cce375bb4a9995d9896)